spec/js/no_in_person_locations_display_spec.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @jest-environment jsdom
+ */
+
+describe('NoInPersonLocationsDisplay', () => {
+  beforeEach(() => {
+    document.body.innerHTML = `
+      <no-in-person-locations-display
```

Comment:
I'm a little confused by how this spec is set up, since this looks like a custom element? How is it actually initializing the React?

... Is it even being run in the test suite?

Reply:
Thanks for posting that Matt. I did ensure it was being ran along with some tests to show I was getting a failure for certain behavior. I followed similar patterns in the repo. It does not look like Jest will run a tsx file (I attempted to adjust jest.config.js testMatch to include tsx files but then get an error for the i18n import in the actual file).